src/sentry/grouping/enhancer/enhancement-configs/mobile@2021-04-02.txt

```diff
@@ -270,7 +275,7 @@ category:malloc +prefix
 category:lock +sentinel +prefix
 category:load +sentinel +prefix
 category:gl +sentinel
-category:gc +sentinel
+category:runtime +sentinel
```

Comment:
We should start adding reasoning in comments to such rules. This one particular, what's the thought behind it? Wouldn't this cause irrelevant runtime frames to be chosen over top app frames in grouping?

Reply:
Fair enough, I added a comment